src/EditorFeatures/CSharp/StringCopyPaste/StringCopyPasteCommandHandler.cs

```diff
@@ -121,16 +121,35 @@ public void ExecuteCommand(PasteCommandArgs args, Action nextCommandHandler, Com
             if (documentBeforePaste == null || documentAfterPaste == null)
                 return;
 
+            // See if we can determine the information about the code the user copied from.
+            var copyPasteService = documentBeforePaste.Project.Solution.Workspace.Services.GetService<IStringCopyPasteService>();
+            if (copyPasteService is null)
+                return;
```

Comment:
`IStringCopyPasteService` is defined and has a default export in the same assembly. It will always be present, so might as well use `GetRequiredService` here.

Reply:
done!